### DISCUSSION TITLE
Version 2 Public view not enabled

### DISCUSSION THREAD

**Discussion by PA4RB** [magicbug/Cloudlog#1678] [Q&A] [answered]
Somehow I've managed to mess things up. The public view is gone. Can't seem to find the place to enable it again. Anyone?

**Comment by phl0** [comment, marked as answer]
Go to you created station logbook and create a "public slug". In the overview you will then find an icon with the publicly accessible link.

**Reply by PA4RB** [reply]
Yes just saw that option. However when using multiple logbooks it's not possible to use one slug to get the total overview of all the logbooks. You have to stick to one logbook only. And I see the map stays blank.

**Reply by phl0** [reply]
You could create one lookbook and link all locations to that. Then you have a station logbook containing all.

**Reply by PA4RB** [reply]
Nice catch Florian....;-) That'll do the trick. The empty map by the way is that something of your concern?

**Reply by magicbug** [reply]
If you grab latest codebase map should show again, it was a change that got missed by me :)
